internal: test-summary fails on Windows

```
--- FAIL: Test (0.03s)
    test-summary_test.go:31: 
        got  Failures (reporting up to 10):
          gocloud.dev\internal\docstore\internal\fields\TestUnexportedAnonymousNonStruct
        ran 6; passed 3; failed 1; skipped 2
        want Failures (reporting up to 10):
          gocloud.dev/internal/docstore/internal/fields/TestUnexportedAnonymousNonStruct
        ran 6; passed 3; failed 1; skipped 2
FAIL
```

--- a/internal/testing/test-summary/test-summary_test.go
+++ b/internal/testing/test-summary/test-summary_test.go
@@ -15,6 +15,8 @@
 package main
 
 import (
+	"fmt"
+	"path/filepath"
 	"strings"
 	"testing"
 )
@@ -24,9 +26,10 @@ func Test(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	want := `Failures (reporting up to 10):
-  gocloud.dev/internal/docstore/internal/fields/TestUnexportedAnonymousNonStruct
-ran 6; passed 3; failed 1; skipped 2`
+	path := filepath.Join("gocloud.dev", "internal", "docstore", "internal", "fields", "TestUnexportedAnonymousNonStruct")
+	want := fmt.Sprintf(`Failures (reporting up to 10):
+  %s
+ran 6; passed 3; failed 1; skipped 2`, path)
 	if got != want {
 		t.Errorf("\ngot  %s\nwant %s", got, want)
 	}

